splink/linker.py

```diff
@@ -1394,18 +1499,21 @@ def cluster_pairwise_predictions_at_threshold(
 
         """
 
-        self._initialise_df_concat_with_tf(df_predict)
+        # Feeding in df_predict forces materiailisation, if it exists in your database
```

Comment:
The first parameter to `_initialise_df_concat_with_tf` is a bool `materialise`.  I think it would be better to make this explicit - e.g.
```
if predict:
    materialise=True
else:
   materialise = False
```
Just feels a bit odd to feed in a SplinkDataFrame that is evaluated as a bool

Reply:
Actually - thinking about this - should we always materialise here?

Does it actually need to be `_initialise_df_concat_with_tf` or can it be `_initialise_df_concat`?